datameta/api/users.py

```diff
@@ -85,56 +85,58 @@ def put(request: Request):
     # Get the targeted user
     target_user = resource_by_id(db, User, user_id)
 
-    if(target_user == None):
+    if target_user is None:
         raise HTTPNotFound() # 404 User ID not found
 
+    has_group_rights = auth_user.group_admin and auth_user.group.uuid == target_user.group.uuid
+    self_changes = auth_user.uuid == target_user.uuid
```

Comment:
```suggestion
    user_is_target = auth_user.uuid == target_user.uuid
```
This naming might be a bit more accurate.